src/style.js

```diff
@@ -143,6 +149,9 @@ export function getStyle(element, property, opt_bypassCache) {
   if (!propertyName) {
     return undefined;
   }
+  if (isVar(propertyName)) {
+    return element.style.getPropertyValue(propertyName);
```

Comment:
Ditto.

Reply:
Moved this to https://github.com/ampproject/amphtml/pull/27145